testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from MDAnalysis.coordinates.GRO import GROReader, GROWriter
+from MDAnalysisTests.coordinates.base import BaseReference, BaseReaderTest, BaseWriterTest
+from MDAnalysisTests.datafiles import COORDINATES_GRO, COORDINATES_GRO_INCOMPLETE_VELOCITY, COORDINATES_GRO_BZ2
+
+
+class GROReference(BaseReference):
+    def __init__(self):
+        super(GROReference, self).__init__()
+        self.trajectory = COORDINATES_GRO
+        self.topology = COORDINATES_GRO
+        self.reader = GROReader
+        self.writer = GROWriter
+        self.ext = 'gro'
+        self.n_frames = 1
+        self.prec = 4
+        self.first_frame.velocities = np.array(
+            [[0.0000, 0.100, 0.200],
+             [0.300, 0.400, 0.500],
+             [0.600, 0.700, 0.800],
+             [0.900, 1.000, 1.100],
+             [1.200, 1.300, 1.400]],
+            dtype=np.float32)
+        self.totaltime = 0
+        self.container_format = True
+
+
+class TestGROReader(BaseReaderTest):
+    def __init__(self, reference=None):
+        if reference is None:
+            reference = GROReference()
+        super(TestGROReader, self).__init__(reference)
+
+
+class TestGROWriter(BaseWriterTest):
+    def __init__(self, reference=None):
+        if reference is None:
+            reference = GROReference()
+        super(TestGROWriter, self).__init__(reference)
+
+
+class GRONoConversionReference(GROReference):
+    def __init__(self):
+        super(GRONoConversionReference, self).__init__()
+        self.first_frame.positions /= 10.0
+        self.first_frame.velocities /= 10.0
+        self.dimensions[:3] /= 10.0
+        self.volume /= 1000
+
+
+class TestGROReaderNoConversion(BaseReaderTest):
+    def __init__(self, reference=None):
+        if reference is None:
+            reference = GRONoConversionReference()
+        super(TestGROReaderNoConversion, self).__init__(reference)
+        self.reader = self.ref.reader(self.ref.trajectory, convert_units=False)
```

Comment:
There are a few tests failing because there isn't a `lowf`/`highf` aux.  The clue to debugging it was that this was limited to the `NoConversion` set of tests.  When you replace `reader` here it doesn't have those auxiliaries.  If you go back to the reference you can see how these are normally added to `self.reader`